apps/client/src/app/[user]/components/LeftSidebar/HydratedStudyList.tsx

```diff
@@ -0,0 +1,17 @@
+import { useMyGroupsQueryObject } from "@/app/api/users/query";
+import { prefetchQuery } from "@/shared/util/prefetch";
+import { HydrationBoundary } from "@tanstack/react-query";
+import StudyList from "./StudyList";
+
+const HydratedStudyList = async () => {
+  const queryOption = useMyGroupsQueryObject();
+  const dehydratedState = await prefetchQuery(queryOption);
```

Comment:
```suggestion
  const dehydratedState = await prefetchQuery(useMyGroupsQueryObject());
```

노션에 컨벤션 적어놓은 것처럼 이런 형식으로 통일해주면 좋을 것 같아용!

Reply:
prefetchQuery를 수정하면서 이곳을 놓쳤었네요. 수정하겠습니다.